### ISSUE TITLE
vli ads

### ISSUE THREAD

**Issue by bnainar** [StevenBlack/hosts#1675]
# Domain(s) to be blocked:
* `assets.vlitag.com`
* `services.vlitag.com`
*  `tag.vlitag.com`
* `static.vliplatform.com`

# Who?
[ValueImpression](https://www.valueimpression.com)
> With more than 8 years of experience in the advertising industry, The ValueImpression has

Definitely a ad company

# Found where?
https://www.javatpoint.com/hadoop-tutorial

**Comment by dnmTX** [comment]
Those most likely originated from `services.vlitag.com`, per my bud AdGuard 😉 : 

![Capture2](https://user-images.githubusercontent.com/34774426/120123235-4a7c6c00-c173-11eb-8187-b8a7045c6b49.PNG)

**Comment by bnainar** [comment]
and it recieves this from that subdomain:

```js
window._VLIOBJ={"state":[],"blk":true,"vinasat":"132kd","ipProbability":0,"mtk":"15083","shn":"services.vlitag.com","tcf":{"tagApi":"viAPItag","tagName":"vitag"}};

(function(doc){try{let url="https://"+window._VLIOBJ.shn+"/uv/?page_url="+encodeURIComponent(window.location.href)+"&mtk="+window._VLIOBJ.mtk;let r=new XMLHttpRequest();

r.responseType="json";r.timeout=5000;r.onreadystatechange=function(){let tname=window._VLIOBJ.tcf.tagName;let tapi=window._VLIOBJ.tcf.tagApi;
if(r.readyState===XMLHttpRequest.DONE && r.status===200){window[tname]=window[tname] || {};
(window[tname].Init=window[tname].Init || []).push(function(){window[tapi].assignConfig(r.response)});}};
r.open("GET",url,true);r.send();}
catch(e){};var s=doc.createElement('script');
s.src='//tag.vlitag.com/v1/1622365212/44613a165a038838f18e827a36e21254.js';
var h=doc.getElementsByTagName('head')[0];h.appendChild(s);
})(document);
```

**Comment by bnainar** [comment]
Does our master list collect from AdGaurd, if so, why wasn't `services.vlitag.com` wasn't blocked by this list? and I can't find it by searching our big hosts file

**Comment by dnmTX** [comment]
It doesn't,rules are complicated there. Wildcard blocking is also possible with it,but not in hosts file.

**Comment by bnainar** [comment]
So, just including those subdomains I provided will be sufficient

**Comment by dnmTX** [comment]
I don't even want to start duscussing QUIC,where the content is delivered by IP and not by (sub)domains. God help us all 🤞

**Comment by bnainar** [comment]
🥲 so what's the future of our list, considering the QUIC protocol? @StevenBlack and @dnmTX

**Comment by dnmTX** [comment]
I was just monitoring that:
IP knocks on the door,dnsmasq does reverse lookup for the record,if all good.door is wide open:
![Capture3](https://user-images.githubusercontent.com/34774426/120124650-fbd2d000-c17a-11eb-95fd-09946c3cac8a.PNG)

....If no record found,shuts the door and that's it:
![Capture4](https://user-images.githubusercontent.com/34774426/120124673-16a54480-c17b-11eb-8066-eb5527f35441.PNG)

So in conclusion,proper records are needed to point to some domain and all is welcome.

**Comment by StevenBlack** [comment]
Thank you for this insight @bnainar these are added in commit 36e70223.  These will be included in the next release.

Sorry for the delay on this one.
